src/main/java/com/practice/splitwise/database/SQLDB.java

```diff
@@ -0,0 +1,118 @@
+package com.practice.splitwise.database;
+
+import com.practice.splitwise.pojo.Transaction;
+import com.practice.splitwise.pojo.UserBean;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SQLDB {
+    private static SQLDB sqldb=new SQLDB();
+    private SQLDB(){ }
+    public static SQLDB getInstance(){
+        return sqldb;
+    }
+    private Map<String,UserBean> userMap=new HashMap<String,UserBean>();
+    private List<Transaction> transactionList = new ArrayList<>();
+    private Map<String, List<Transaction>> paidByUserMap = new HashMap<>();
+    private Map<String, List<Transaction>> paidForUserMap = new HashMap<>();
+
+    public String addUsers(List<UserBean> userList) {
+        if(userMap==null)
+            userMap=new HashMap<String,UserBean>();
+        StringBuilder sb1 = new StringBuilder();
+        StringBuilder sb2 = new StringBuilder();
+        for(UserBean userBean:userList){
+            if(userMap.get(userBean.getId())==null) {

```

Comment:
Will result in null pointer exception if userBean.getId() is null? we should put validations in service layer itself.